ee/tabby-db/src/repositories.rs

```diff
@@ -64,6 +64,13 @@ impl DbConn {
             Err(anyhow!("failed to update: repository not found"))
         }
     }
+
+    pub async fn get_repository_git_url(&self, name: String) -> Result<String> {
```

Comment:
```suggestion
    pub async fn get_repository_by_name(&self, name: String) -> Result<RepositoryDAO> {
```

Also make sure we add an index on `name`